SU2_CFD/src/integration_time.cpp

```diff
@@ -845,7 +867,29 @@ void CSingleGridIntegration::SingleGrid_Iteration(CGeometry ****geometry, CSolve
   /*--- Convergence strategy ---*/
   
   Convergence_Monitoring(geometry[iZone][iInst][FinestMesh], config[iZone], Iteration, monitor, FinestMesh);
-  
+
+    /* --- If fixed C_L mode, ensure that C_L is converged to target C_L --- */
+  if (config[iZone]->GetFixed_CL_Mode() && Convergence){
```

Comment:
See comment above. Same here.

Reply:
This was a good point and it has been addressed, details in comment below.